docs/en/guides/python-3-best-practices.md

```diff
@@ -12,11 +12,11 @@ lang: en
 
 There are a few compelling reasons to make the leap to Python 3.  Perhaps the most dramatic is the Python 2 end of life, which occurred on January 1, 2020[^1]. With the sunsetting of Python 2, all support for Python 2 ceases, meaning that even new security vulnerabilities found in Python 2 will not be addressed.
 
-For CY2020, the [VFX reference platform](https://vfxplatform.com/) makes the switch as well, targeting Python version 3.7.x.  As a practical matter for many of us, all of this will mean we don't have much choice in when to add support for Python 3 -- as DCCs (digital content creation applications) that we develop for begin to move to Python 3 interpreters, it will become a necessity to support them.
```

Comment:
Problem here. The change is valid for SGD version 1.7. But version 1.8 follows CY2022.